src/components/QueryResult.jsx

```diff
@@ -33,7 +33,7 @@ const QueryResult = (props) => {
             case "family":
                 columns = ["score", "pctId", "aln"];
                 props.dataPromise.then((data) => {
-                    data = data.slice(0, 20);
+                    data = data.items.slice(0, 20);
```

Comment:
we can get rid of `.slice(0, 20)` here - `itemsPerPage` is already defined.

This was for when 100 results were returned and the D3 view box is only configured to display 20 max